### PR TITLE
Add a warning when it is detected that a process is uwsgi based but u…

### PR DESCRIPTION
**Pull request by dannon**
…wsgidecorators is not available.

### CODE REVIEW

File: lib/galaxy/util/postfork.py [hunk 1]
```diff
@@ -19,6 +19,10 @@
     def pf_dec(func):
         return func
     postfork = pf_dec
+    if process_is_uwsgi:
+        print("WARNING: This is a uwsgi process but the uwsgidecorators library"
+              " is unavailable.  This is likely due to using an external (not"
+              " in Galaxy's .venv) uwsgi and you may experience errors.")
```
Comment (natefoo) [inline lib/galaxy/util/postfork.py]: Could this just be "virtualenv" instead of ".venv"?

Comment (dannon) [inline lib/galaxy/util/postfork.py]: Yeah.

**Comment by natefoo** [comment]
We've got the biggest, best commit messages, people.

**Comment by dannon** [comment]
Yuge commit messages are overrated.

**Comment by martenson** [comment]
Should build some protection around them or sth...

**Pull request merged**